src/materialized/src/bin/materialized/main.rs

```diff
@@ -475,6 +486,25 @@ fn run(args: Args) -> Result<(), anyhow::Error> {
         let key = args.tls_key.unwrap();
         Some(materialized::TlsConfig { mode, cert, key })
     };
+    let mzcloud_params = if let Some(tenant_id) = args.mz_cloud_tenant {
+        let frontegg_jwk = match args.mz_cloud_frontegg_jwk {
+            None => bail!("expected mz_cloud_frontegg_jwk"),
+            // Attempt to base64 decode the jwk, otherwise keep it as is. This is
+            // unambiguous because the standard base64 alphabet does not contain dash,
+            // which is present in the key.
+            Some(jwk) => base64::decode(&jwk).unwrap_or_else(|_| jwk.as_bytes().to_vec()),
+        };
+        Some(MzCloudParams {
+            admin_api_token_url: args.mz_cloud_api_token_url.unwrap_or_else(|| {
+                "https://admin.cloud.materialize.com/identity/resources/auth/v1/api-token"
```

Comment:
Hrmmmmm, I'm not loving the hardcoded URL - this makes it impossible to authenticate users in staging / dev stacks. Could we make this configurable via a commandline parameter?

Reply:
Oh, I seeeeee. In that case, I think we can drop that URL - it's unlikely to be real anyway (:

Reply:
Drop as in...make this a required instead of optional parameter?

Reply:
Yeah, sorry for being unclear. I think it doesn't make sense to provide a default URL here - making it mandatory would be the right thing to do, I think.